src/main/scala/scorex/core/newserialization/ByteStringWriter.scala

```diff
@@ -0,0 +1,128 @@
+package scorex.core.newserialization
+
+import java.nio.ByteOrder
+import java.util.{Arrays, BitSet}
+
+import akka.util.ByteString
+
+class ByteStringWriter extends ScorexWriter {
+  override type CH = ByteString
+  private implicit val byteOrder = ByteOrder.BIG_ENDIAN
+
+  override def newWriter(): ScorexWriter.Aux[CH] = {
```

Comment:
@VictorMikheev I'm quite confused with why do we need `newWriter()` at all. And `putChunk`, `putByteString2` don't clear things up.

Reply:
newWriter creates instance of Writer with same inner type of data.
ByteString2 deleted